front/tests/pages/operational-studies/paced-train-section.ts

```diff
@@ -183,20 +187,36 @@ class PacedTrainSection extends CommonPage {
     await expect(occurrenceArrivalTimeLocator).toHaveText(expectedArrivalTime);
   }
 
-  private async getActionButtonsLocators(
-    itemIndex: number,
-    itemType: 'paced-train' | 'occurrence'
-  ): Promise<Record<string, Locator>> {
+  async getActionButtonsLocators({
+    itemIndex,
+    itemType,
+    withExceptions = false,
+  }: {
+    itemIndex: number;
+    itemType: 'paced-train' | 'occurrence';
+    withExceptions?: boolean;
+  }): Promise<Record<string, Locator>> {
     const timetableItem =
       itemType === 'paced-train'
```

Comment:
you can create a const and use it in the rest of the function to improve readability
```suggestion
    const isPacedTrain = itemType === 'paced-train';
```

Reply:
Done